server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -1041,7 +1042,9 @@ public String toString() {
                 org.elasticsearch.ingest.IngestProcessorException.class,
                 org.elasticsearch.ingest.IngestProcessorException::new,
                 157,
-                Version.V_7_5_0);
+                Version.V_7_5_0),
+        FS_HEALTH_CHECK_FAILURE_EXCEPTION(FsHealthCheckFailureException.class,
```

Comment:
Please format this like the other declarations:

- multiple lines
- fully-qualified class names rather than an `import`
- no trailing comma